salt/_modules/metalk8s_network.py

```diff
@@ -308,6 +308,9 @@ def get_control_plane_ingress_external_ips():
             f"Unable to get master Control Plane IPs: {mine_ret}"
         )
 
-    return [__salt__["metalk8s_network.get_control_plane_ingress_ip"]()] + sorted(
-        list(mine_ret.values())
-    )
+    control_plane_ip = __salt__["metalk8s_network.get_control_plane_ingress_ip"]()
```

Comment:
```suggestion
    control_plane_ingress_ip = __salt__["metalk8s_network.get_control_plane_ingress_ip"]()
```

It's important it's not necessarily a control plane IP